doc/nrf/app_dev/device_guides/hpf/hpf_asm.drawio.html

```diff
@@ -0,0 +1,11 @@
+<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=5,IE=9" ><![endif]-->
+<!DOCTYPE html>
```

Comment:
Rework in Visio: https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120293046/Figure+guide